### ISSUE TITLE
RSS feed generation

### ISSUE THREAD

**Issue by leouieda** [jandecaluwe/urubu#45]
First of all, thank you for the great engine!

Is there any way to generate RSS feeds? I'd like to have feeds for each folder in my site. 
If not, is there any plan of implementing this?

I could give a hand but I have no idea how these things are done.

**Comment by jandecaluwe** [comment]
I am labeling this for `help wanted`, as I have little time to look into this.

Are you in Rio?

**Comment by leouieda** [comment]
OK, no problem. I'll look into this when I have some spare time.

Yes I am! I work at the state University (UERJ).

**Comment by jandecaluwe** [comment]
Que saudade ;-) May visit shortly. Can you recommend a good churrascaria over there these days?

**Comment by leouieda** [comment]
The most famous that I know of is "Porcão". I've never been myself but many friends say that it's awesome. The [one in Flamengo](https://www.google.com.br/maps/place/Porc%C3%A3o+Rio's/@-22.9500121,-43.2529712,11.75z/data=!4m5!1m2!2m1!1sporcao+churrascaria!3m1!1s0x0000000000000000:0x84a9eb6aeb695758?hl=en) has a particularly nice view of the Pão de Açúcar.

**Comment by jandecaluwe** [comment]
Yes, that's the famous one isn't. At some point I read somewhere about a meat restaurant where Tom Jobim liked to go at the time, I think somewhere near the Lagoa, but I don't remember the name and I don't know if it's still there.

Anyway, for the record, a churrascaria is not what I associate most with Brasil (in terms of dining that would be more sophisticated restaurants llike DOM and Mani in São Paulo and Roberta Sudbrack in Rio, but most of all it would be  that would be the fantastic music culture) but from time to time one has to do it "once" ;-)

**Comment by leouieda** [comment]
There's lots to like (and dislike) about Rio and Brasil in general. But it's a nice place to live if you have a moderate income. Rio is very expensive.

The [Esplanada Grill](http://esplanadagrill.com.br/) is pretty famous in Ipanema. And I've heard a lot about [CT Boucherie](http://ctboucherie.com.br/) as well.

**Comment by jandecaluwe** [comment]
I know - having lived in São Paulo for two years.

Thanks for the links, I will definitely check them out.

While googling I also found the name of Tom Jobim's favorite restaurant: Plataforma. But I also read it is closed now and may never re-open.

**Comment by jandecaluwe** [comment]
[Fogo de Chão](http://www.fogodechao.com.br/rio-de-janeiro-%E2%80%A2-rj/) em Botafogo!

**Comment by leouieda** [comment]
Have you been there?

**Comment by jandecaluwe** [comment]
Yes, recommended by the hotel - very nice churrascaria.

**Comment by leouieda** [comment]
It's a famous chain here but I've never been there myself. I'm much for churrasco, except when it's at home.

**Comment by twinshadow** [comment]
My current approach is to modify `urubu/processors.py`, given an index.md file with `feed: true` in the front-matter. This would generate the feed.xml as a side-effect of processing index.md, which may not be desired. The issue of project output files and templates both assuming `html` for everything could use some refactoring.

**Comment by 42sol-eu** [comment]
maybe we should add this {{The issue of project output files and templates both assuming html for everything could use some refactoring.}} an issue

**Comment by 42sol-eu** [comment]
as mentioned in my other issue see https://github.com/42sol-eu/jinja2-rss-generator for my approach. If you want to be involved feel free to do so.

**Comment by Clarin98** [comment]
This project https://github.com/thp/urlwatch with rss support could be a good alternative for the issue
